src/modules/drawing/components/DrawingTooltip.vue

```diff
@@ -34,113 +44,149 @@ export default {
+            const drawingLayer = this.getDrawingLayer()
 
-                    // keeping track that features are being overed (even if not selected)
+            // detecting features under the mouse cursor
+            let hoveringSelectableFeature = false
+            let hoveringSelectedFeature = false
+
+            const hasFeatureSelected = this.selectedFeatures.length > 0
+            // we only keep track of the first feature's info (the one on top of the stack)
+            const selectedFeatureId = hasFeatureSelected
+                ? this.selectedFeatures[0]?.id.replace('drawing_feature_', '')
+                : null
+            let featureUnderCursor
+
+            map.forEachFeatureAtPixel(
+                event.pixel,
+                (feature) => {
+                    // Keeping track that features are being hovered (even if not selected)
                     hoveringSelectableFeature = true
 
                     const isSelectedFeature =
-                        hasFeatureSelected &&
-                        selectedFeatureId === `drawing_feature_${feature.getId()}`
+                        hasFeatureSelected && selectedFeatureId === feature.getId()
 
                     hoveringSelectedFeature = hoveringSelectedFeature || isSelectedFeature
                     // subsequent features will be ignored as featureUnderCursor will already be set
                     if (!featureUnderCursor || isSelectedFeature) {
                         featureUnderCursor = feature
                     }
+                },
+                {
+                    layerFilter: (layer) => layer === drawingLayer,
+                    hitTolerance,
+                }
+            )
+
+            let featureType = featureUnderCursor?.get('drawingMode')?.toLowerCase()
+            let translationKeys
+
+            if (hoveringSelectedFeature) {
+                mapElement.classList.add(cssGrab)
+                mapElement.classList.remove(cssPointer)
+
+                let hoveringVertex = getVertexCoordinates(featureUnderCursor).some((coordinate) => {
+                    let pixel = map.getPixelFromCoordinate(coordinate)
+                    return pointWithinTolerance(pixel, event.pixel, hitTolerance)
                 })
-                // detecting if we are hovering the area of a polygon (and not its borders)
-                // if so, we suppress the flag "hoveringSelectedFeature" as it is of no value
-                // to know that we are hovering the area (only borders have interactivity)
-                if (featureUnderCursor?.getGeometry() instanceof Polygon) {
-                    hoveringSelectedFeature = false
+
+                // Display a help tooltip when modifying
+                if (hoveringVertex || ['marker', 'annotation'].includes(featureType)) {
+                    translationKeys = `modify_existing_vertex_${featureType}`
+                } else {
+                    translationKeys = `modify_new_vertex_${featureType}`
```

Comment:
Here I would prefer that we lower the case of `featureType` when we build our string, and then use `EditableFeatureType` enum in the `.includes` (it will be more resilient in case of refactoring)

Reply:
What do you mean by "lower the case"?

Reply:
Ok

Reply:
Ok